README.md

```diff
@@ -27,6 +27,7 @@ git clone https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation.git
 cd OpenAssetIO-MediaCreation
 python -m venv .venv
 . ./.venv/bin/activate
+python -m pip install .
```

Comment:
[question] Good idea, I wonder if `openassetio-mediacreation` should be removed from `examples/resources/requirements.txt` then, to avoid potential confusion?

As far as a quick google can tell, relative paths are relative to the cwd in requirements.txt, which is unfortunate.  If paths could be relative to the requirements file itself then we could have `../..` in there.

Reply:
Yeah I was wondering that, but at the same time, I was hoping that that `requirement.txt` file would help notebook hosting services set the env up, without needing as much custom work. I _think_ It's already going to be a pain with Binder as it's not at the root.

Reply:
(I think its moot too as BAL is going to have an explicit dependency on media creation soon anyway....)